go/client/cmd_ctl_restart.go

```diff
@@ -55,7 +55,7 @@ func (s *CmdCtlRestart) Run() error {
 
 	// If the watchdog or launchd started this process, it will do the restarting.
 	// Otherwise we have to.
-	if config.ForkType != keybase1.ForkType_WATCHDOG && config.ForkType != keybase1.ForkType_LAUNCHD {
+	if config.ForkType != keybase1.ForkType_WATCHDOG && len(config.Label) > 0 {
```

Comment:
Please let me look at this before it goes in. I am away from the computer
until about 8. I am sorry

On Thursday, January 7, 2016, Steve Sanders <notifications@github.com>
wrote:

> In go/client/cmd_ctl_restart.go
> <https://github.com/keybase/client/pull/1660#discussion_r49139672>:
>
> > @@ -55,7 +55,7 @@ func (s *CmdCtlRestart) Run() error {
> >
> >  	// If the watchdog or launchd started this process, it will do the restarting.
> >  	// Otherwise we have to.
> > -	if config.ForkType != keybase1.ForkType_WATCHDOG && config.ForkType != keybase1.ForkType_LAUNCHD {
> > +	if config.ForkType != keybase1.ForkType_WATCHDOG && len(config.Label) > 0 {
>
> Is this the right thing to test, or are we looking for a specific label
> here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1660/files#r49139672>.
>